src/stratigraphy/groundwater/groundwater_extraction.py

```diff
@@ -137,9 +141,11 @@ class GroundwaterLevelExtractor(DataExtractor):
     feature_name = "groundwater"
 
     # look for elevation values to the left, right and/or immediately below the key
-    search_left_factor: float = 2
+    search_left_factor: float = 3  # NOTE: check files 267125334-bp.pdf, 267125338-bp.pdf, and 267125339-bp.pdf if this
+    # value is too high, as it might lead to false positives
```

Comment:
Yes, this, in combination with the new `search_above_factor`, indeed seems to lead to too many false positives. But maybe the ongoing work in https://jira.swisstopo.ch/browse/LGVISIUM-77 will already make this more robust again?

Reply:
The issue I was facing with the 267125334-bp.pdf, 267125338-bp.pdf, and 267125339-bp.pdf bore profiles was that False Positives were generated if the left search factor was too large. In these profiles, the algorithm would find the depth column and extract data. 

I think the best option performance-wise would be to use the default values from the `main` branch.